html/elements/form.json

```
@@ -349,6 +349,38 @@
             }
           }
         },
+        "rel": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "107"
```

In #18431, the matching interface property was set to "108" instead.  ChromeStatus has been pretty inaccurate lately, and I trust the existing data that comes from our own tests more:

```suggestion
                "version_added": "108"
```